colorbleed/plugins/global/publish/submit_publish_job.py

```diff
@@ -134,8 +136,10 @@ def process(self, instance):
         # Get a submission job
         job = instance.data.get("deadlineSubmissionJob")
         if not job:
-            raise RuntimeError("Can't continue without valid deadline "
-                               "submission prior to this plug-in.")
+            self.log.warning("Can't continue without valid deadline "
```

Comment:
@aardschok Why shouldn't this be a raised error? :) Since it is actually failing completely.